aiohttp/web_exceptions.py

```diff
@@ -241,15 +297,22 @@ def __init__(self,
                  *,
                  headers: Optional[LooseHeaders]=None,
                  reason: Optional[str]=None,
-                 body: Any=None,
                  text: Optional[str]=None,
                  content_type: Optional[str]=None) -> None:
         allow = ','.join(sorted(allowed_methods))
         super().__init__(headers=headers, reason=reason,
-                         body=body, text=text, content_type=content_type)
+                         text=text, content_type=content_type)
         self.headers['Allow'] = allow
-        self.allowed_methods = set(allowed_methods)  # type: Set[str]
-        self.method = method.upper()
+        self._allowed = set(allowed_methods)  # type: Set[str]
```

Comment:
Should `allowed_methods` be uppercased as well?

Reply:
Good question.
HTTP verbs are case sensitive by RFC.
Maybe even `method.upper()` is redundant.
aiohttp dispatcher never raises the lowercased method.